bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -58,6 +58,30 @@
 				<description>JSON config that allows to modify the media state detection strategy for each app. Refer to the binding
 					documentation.</description>
 			</parameter>
+			<parameter name="volumeSettingKey" type="text" required="true">
+				<label>Volume Setting Key</label>
+				<description>Settings key for android versions where the volume level is gathered using the 'settings' cli
+					(>=android 11).</description>
+				<default>volume_music_hdmi</default>
+				<options>
+					<option value="volume_bluetooth_sco">volume_bluetooth_sco</option>
+					<option value="volume_music">volume_music</option>
+					<option value="volume_music_bt_a2dp">volume_music_bt_a2dp</option>
+					<option value="volume_music_hdmi">volume_music_hdmi</option>
+					<option value="volume_music_headphone">volume_music_headphone</option>
+					<option value="volume_music_headset">volume_music_headset</option>
+					<option value="volume_music_headset">volume_music_headset</option>
+					<option value="volume_music_usb_headset">volume_music_usb_headset</option>
+					<option value="volume_system">volume_system</option>
```

Comment:
If you define options, at least some of the options should have a label different from its value. If not, the options are useless.
I would suggest to not use _ inside the labels.